.github/workflows/pr.yml

```diff
@@ -0,0 +1,14 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
This configuration will only build Pull Requests that have `master` as base.

I think we may want to build _any_ pull request, regardless of whether `master` is the base. Additionally, I think it may be a good idea to run tests for any `push` to `master`. To do this, consider the following configuration:

```yaml
on:
  push:
    branches:
      - master
  pull_request:
```

If we end up building on pushes to master too, then we may want to use a more generic name for this workflow, e.g. "Tests" as opposed to "Pull Request".

Reply:
That makes sense to me. I couldn't find an "all" option for branches, so instead just specified the types - open/synchronize - which I believe should address this.

Reply:
If you omit `types`, then I believe it should build all events. At least that's what we've been doing for [sparksteps](https://github.com/jwplayer/sparksteps/blob/master/.github/workflows/tests.yml#L6).

Reply:
Ah I had thought that wasn't a valid config. Either way, I'll stick with being explicit for now. Thanks for the review.